extension/src/main/scala/io/scalac/mesmer/extension/upstream/OpenTelemetryDispatcherMetricsMonitor.scala

```diff
@@ -0,0 +1,100 @@
+package io.scalac.mesmer.extension.upstream
+
+import com.typesafe.config.Config
+import io.opentelemetry.api.metrics.Meter
+import io.scalac.mesmer.core.config.MesmerConfiguration
+import io.scalac.mesmer.core.module.AkkaDispatcherModule
+import io.scalac.mesmer.extension.metric.HttpMetricsMonitor.BoundMonitor
+import io.scalac.mesmer.extension.metric.{DispatcherStaticMetricsMonitor, MetricObserver, RegisterRoot}
+import io.scalac.mesmer.extension.upstream.OpenTelemetryDispatcherMetricsMonitor.MetricNames
+import io.scalac.mesmer.extension.upstream.opentelemetry.{GaugeBuilderAdapter, SynchronousInstrumentFactory}
+
+object OpenTelemetryDispatcherMetricsMonitor {
+
+  final case class MetricNames(
+    executor: String,
+    minThreads: String,
+    maxThreads: String,
+    parallelismFactor: String
+  )
+
+  object MetricNames extends MesmerConfiguration[MetricNames] {
+    val defaultConfig: MetricNames =
+      MetricNames("akka_dispatcher_executor", "akka_dispatcher_min_threads", "akka_dispatcher_max_threads", "akka_dispatcher_parallelism_factor")
+
+    protected val mesmerConfig: String = "metrics.dispatcher-metrics"
+
+    protected def extractFromConfig(config: Config): MetricNames = {
+
```

Comment:
which config is getting passed here? Is it the akka config from application.conf ?

Reply:
I think that you should be able to read config values from application.conf and reference.conf files using that method. I'm not sure if this works well - I once created an issue about this: https://github.com/ScalaConsultants/mesmer-akka-agent/issues/288 

@worekleszczy can you confirm? 

Either way, don't worry about it too much now - you can leave everything enabled for now and focus on delivering functioning dispatcher metrics. Config is currently being worked on by @worekleszczy so it will change anyway. We can add some simple configurations after the changes made by @worekleszczy are merged.